src/temaWeek8Exception/main/Repository.java

```diff
@@ -0,0 +1,101 @@
+package temaWeek8Exception.main;
+
+import java.time.Year;
+import java.util.*;
+
+public class Repository {
+	String firstName = "";
+	String lastName = "";
+	String dateOfBirth = "";
+	String gender = "";
+	String ID = "";
+	
+	
+	Student student = new Student(firstName, lastName, dateOfBirth, gender, ID);
```

Comment:
It might be difficult to manage all changes if you use this `student` property... I think there should be a better design here.